test/unit/data.js

```diff
@@ -874,3 +874,18 @@ test( "Check that the expando is removed when there's no more data", function()
 		}
 	}
 });
+
+test( ".data(prop) does not create expando", function() {
+	expect( 2 );
+
+	var div = jQuery( "<div/>" );
+
+	div.data("foo");
+	equal( false, jQuery.hasData( div[0] ) );
+	// Make sure no expando has been added
+	for ( key in div[ 0 ] ) {
```

Comment:
> at the top.

We have an example of something like that in the guide https://contribute.jquery.org/style-guide/js/#spacing, but strictly that is not a requirement anymore